idunn/utils/circuit_breaker.py

```diff
@@ -24,11 +24,45 @@ def state_change(self, cb, old_state, new_state):
 
 
 class IdunnCircuitBreaker(pybreaker.CircuitBreaker):
-    def __init__(self, name):
+    def __init__(self, name, fail_max, reset_timeout):
         super().__init__(
-            fail_max=settings["CIRCUIT_BREAKER_MAXFAIL"],
-            reset_timeout=settings["CIRCUIT_BREAKER_TIMEOUT"],
+            fail_max=fail_max,
+            reset_timeout=reset_timeout,
```

Comment:
I was just wondering about this while reviewing your other PR, that's not the first time I forget about this ^^'

Reply:
Although, this should probably be the responsibility of the caller? It makes more sense to take an int than a string that will be parsed? (But maybe I'm wrong since there is no clean way to enforce the input type)

Reply:
It's not very important here, as we can easily find and adapt all `__init__` calls, if necessary.
But as a rule of thumb, I would apply the robustness principle: "Be conservative in what you send, be liberal in what you accept". In such a case, accepting a string in addition to int wouldn't do any harm, I think.